pkg/policies/loader.go

```diff
@@ -200,3 +237,12 @@ func refParts(ref string) (string, string) {
 	}
 	return "", parts[0]
 }
+
+func policyReferenceResourceDescriptor(ref string, digest crv1.Hash) *v12.ResourceDescriptor {
```

Comment:
it's weird that we need to import protos from Attestation API for ResourceDescriptor. We might want to extract it in a follow up.

Reply:
I think you are right, will change that